webapp/src/containers/MasternodesPage/components/MineNodeFooter/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { I18n } from 'react-redux-i18n';
+import { Row, Col } from 'reactstrap';
+import { MasterNodeObject } from '../../masterNodeInterface';
+
+interface MineNodeFooterProps {
+  enabledMasternodes: MasterNodeObject[];
+}
+
+const MineNodeFooter: React.FunctionComponent<MineNodeFooterProps> = (
+  props: MineNodeFooterProps
+) => {
+  const { enabledMasternodes } = props;
+
+  return (
+    <Row>
+      <Col md='2'>
+        <span>
+          <small className='text-muted'>
+            {I18n.t('containers.masterNodes.masterNodesPage.collateralsActive')}
+          </small>
+        </span>
+        <div className='d-flex align-items-center'>
+          <span className={`txn-status-enabled`}></span>
+          <span className='ml-2'>{enabledMasternodes.length}</span>
+        </div>
+      </Col>
+      {/* <Col md='2'>
```

Comment:
Remove if not needed

Reply:
I usually ask to remove comments but we plan to renable this in the next update so I think this is ok here

Reply:
It's in design but commented for now maybe it is useful in the next release.